### ISSUE TITLE
Units in TRestDetectorGas

### ISSUE THREAD

**Issue by nkx111** [rest-for-physics/detectorlib#20]
I think we should make a same system of units for TRestDetectorGas. Now the diffusion units are cm^0.5 and the e-Field units are V/mm. Moreover the plotting methods receive input field in V/cm. This is very confusing. Maybe we can make all the *distance* field in unit mm, as in REST standard unit system.

**Comment by nkx111** [comment]
The conversion is supported from TRestSystemOfUnits that `1*(cm^0.5)==0.316*(mm^0.5)`. I think it is easy to convert diffusion parameters to standard unit.

**Comment by jgalan** [comment]
Right, it is confusing, but for me it is strange to express field and diffusion in `V/mm` and `mm^0.5`. I am pretty much used to `V/cm` and `cm^0.5`, for those parameters. So it could be that units are stored internally in `mm` and `V/mm` but we operate them in `V/cm`? Just as it is done at the plotting methods.

Perhaps @cmargalejo or @juanangp got any insights here

**Comment by cmargalejo** [comment]
In my opinion it should be compulsory for the user to include the units and then REST can internally convert them to the standard unit system if required. I think that would avoid a lot of problems. In case this is not possible, I would keep the same units everywhere even if we are not used to them, so that any new user (and experienced user!) knows exactly what to expect.
